templates/slurm/slurm.yaml

```diff
@@ -0,0 +1,66 @@
+{{ if .Values.slurm.deploy }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "cytomine.name" . }}-slurm
+  labels:
+    role: slurm
+    app: {{ include "cytomine.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: {{ .Release.Name }}-slurm
+    release: {{ .Release.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ template "cytomine.name" . }}-slurm
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        role: slurm
+        app: {{ template "cytomine.name" . }}-slurm
+        release: {{ .Release.Name }}
+    spec:
+      hostname: slurm
+      serviceAccountName: {{ include "cytomine.serviceAccountName" . }}
+      volumes:
+        - name: singularity-images
+          persistentVolumeClaim:
+            claimName: {{ template "cytomine.name" . }}-singularity-claim
+        - name: slurm-config
+          configMap:
+            name: {{ include "cytomine.name" . }}-slurm-config
+      containers:
+        - name: {{ .Chart.Name }}-slurm
+          image: "{{ .Values.slurm.image.repository }}:{{ .Values.slurm.image.tag }}"
+          imagePullPolicy: {{ .Values.slurm.image.pullPolicy }}
+          volumeMounts:
+            - mountPath: "/data/softwares/images"
+              name: singularity-images
+            - name: slurm-config
+              mountPath: /home/cytomine/.ssh/authorized_keys
+              subPath: authorized_keys
+            - name: slurm-config
+              mountPath: /etc/slurm/slurm.conf
+              subPath: slurm.conf
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+            - name: ssh
+              containerPort: 22
+              protocol: TCP
```

Comment:
This will require the pod to run stuff as UID < 1000

Reply:
yeah - I don't run slurm anymore, so I should probably just get rid och this and the default software router!